functions-and-scope.js

```diff
@@ -42,6 +73,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6.642857142857143
 
+// Een gemiddelde wordt berekend door alle cijfers bij elkaar op te tellen en dat vervolgens te delen door het aantal cijfers.
+// We willen dus alle cijfers bij elkaar optellen en dat vervolgens delen door de lengte van de array.
+// Hier gebruiken we weer een for loop voor om door alle cijfers heen te kunnen lopen.
+
+function averageGrade(grades) {
+    let counter = 0;
+    let total = 0;
+    for (let i = 0; i < grades.length; i++) {
+        total += grades[i];
+        counter++;
+    }
+    const average = total / counter;
```

Comment:
Je hoeft geen counter bij te houden, omdat die altijd gelijk zal zijn aan `grades.length`
```suggestion
    const average = total / grades.length;
```